lib/iex/test/iex/helpers_test.exs

```diff
@@ -49,6 +49,22 @@ defmodule IEx.HelpersTest do
     assert capture_io(fn -> h __info__ end) == "No documentation for __info__ was found\n"
   end
 
+  test "h helper for callbacks" do
+    with_file ["a_behaviour.ex", "impl.ex"], [behaviour_module, impl_module], fn ->
```

Comment:
This didn't work because exs scripts don't compile docs...

```
  1) test h helper for callbacks (IEx.HelpersTest)
     test/iex/helpers_test.exs:67
     Assertion with == failed
     code: capture_io(fn -> h(Impl.first() / 1) end) == "* @callback first(integer()) :: integer()\n\nDocs for ABehaviour.first\n"
     lhs:  "IEx.HelpersTest.Impl was not compiled with docs\n"
     rhs:  "* @callback first(integer()) :: integer()\n\nDocs for ABehaviour.first\n"
     stacktrace:
       test/iex/helpers_test.exs:68
```

Is there any other better way? I didn't find any actual implementations of `Behaviours` in the Elixir source that we could use...